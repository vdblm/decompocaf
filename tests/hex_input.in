01234
0x1f0a
0X12Fa
-0xff10a
+0X1f12
