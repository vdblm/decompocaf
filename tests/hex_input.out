1234
7946
4858
-1044746
7954
